core/src/main/kotlin/org/evomaster/core/problem/rest/OpenApiAccess.kt

```diff
@@ -36,23 +40,33 @@ object OpenApiAccess {
                 ?: throw SutProblemException("Failed to parse OpenApi schema: " + parseResults.messages.joinToString("\n"))
     }
 
-    fun getOpenAPIFromURL(openApiUrl: String): OpenAPI {
+    fun getOpenAPIFromURL(openApiUrl: String, authentication : HttpWsAuthenticationInfo = HttpWsNoAuth() ): OpenAPI {
 
         //could be either JSON or YAML
        val data = if(openApiUrl.startsWith("http", true)){
-           readFromRemoteServer(openApiUrl)
+           readFromRemoteServer(openApiUrl, authentication)
        } else {
            readFromDisk(openApiUrl)
        }
 
+        // if the swagger could not be retrieved because of unauthorized access, return an empty OpanAPI
+        if (data == unauthorizedSchemaAccess) {
+            return OpenAPI()
```

Comment:
no. if there is a problem, shouldn't silently hide it. rather throw a `SutProblemException` with meaningful error message